cardea-data/src/main/java/ca/on/oicr/gsi/cardea/data/ShesmuSample.java

```diff
@@ -16,29 +13,36 @@ public class ShesmuSample {
 
   private final String id;
   private final boolean supplemental;
+  private final boolean qcFailed;
 
 
   private ShesmuSample(Builder builder) {
     this.id = requireNonNull(builder.id);
     this.supplemental = requireNonNull(builder.supplemental);
+    this.qcFailed = requireNonNull(builder.qcFailed);
   }
 
 
   public String getId() {
     return id;
   }
 
-  public Boolean isSupplemental() {
+  public boolean isSupplemental() {
     return supplemental;
   }
 
+  public boolean isQcFailed() {
+    return qcFailed;
+  }
+
 
 
   @JsonPOJOBuilder(withPrefix = "")
   public static class Builder {
 
     private String id;
     private boolean supplemental;
+    private Boolean qcFailed;
```

Comment:
I realize it wasn't added in this PR, but `supplemental` should be `Boolean` in the builder too